examples/webgl_buffergeometry_instancing.html

```diff
@@ -169,7 +169,7 @@
 			renderer.setSize( window.innerWidth, window.innerHeight );
 			container.appendChild( renderer.domElement );
 
-			if ( renderer.extensions.get( 'ANGLE_instanced_arrays' ) === null ) {
+			if ( renderer.capabilities.isWebGL2 === false && renderer.extensions.get( 'ANGLE_instanced_arrays' ) === null ) {
```

Comment:
I personally think it's better to add `renderer.capabilities.isWebGL2 === false` instead of completely removing the extension check.

Reply:
Yes, I think you're right. Better to keep them, I was being too lazy yesterday 😅